benefits/eligibility/templates/eligibility/includes/media-item--identification--start--login-gov.html

```diff
@@ -7,21 +7,17 @@
 {% endblock heading %}
 
 {% block body %}
-  <div class="media-body--details">
-    <p>
-      {% translate "You will be able to create an account using your email address if you do not already have one. We use your Login.gov account to verify your identity." %}
-      {% translate "Learn more about identity verification" as text %}
-      {% include "core/includes/modal-trigger.html" with modal="modal--identity-verification" text=text period=True %}
-    </p>
-    <div class="media-body--items">
-      <p>{% translate "For this process you will need:" %}</p>
-      <ul>
-        <li>{% translate "Your state-issued ID card" %}</li>
-        <li>{% translate "Your Social Security number" %}</li>
-        <li>{% translate "A phone number with a phone plan associated with your name" %}</li>
-      </ul>
-    </div>
-  </div>
+  <p>
+    {% translate "You will be able to create an account using your email address if you do not already have one. We use your Login.gov account to verify your identity." %}
+    {% translate "Learn more about identity verification" as text %}
+    {% include "core/includes/modal-trigger.html" with modal="modal--identity-verification" text=text period=True %}
+  </p>
+  <p>{% translate "For this process you will need:" %}</p>
+  <ul class="list-disc ms-lg-5 ms-0">
```

Comment:
Create a new utility class, `list-disc`, which turns an unordered list's bullets into the `disc` shape. Follows Bootstrap's style (`list-unstyled`).

`ms-lg-5 ms-0`: Adds a large margin-left on Desktop and none on Mobile, to make the unordered list look indented.